packages/@dcl/ecs/src/index.ts

```diff
@@ -15,3 +16,13 @@ export * from './runtime/types'
 export * from './engine/events'
 export * from './runtime/observables'
 export * from './runtime/temp-fp/Observable'
+
+export type EcsElements = {
+  entity: unknown
+}
```

Comment:
This one is duplicated in `@dcl/ecs` I think this one is the final and the other should be removed